.codespell-exclude-lines

```diff
@@ -0,0 +1,34 @@
+            elif getargbool 0 ro; then
```

Comment:
I am afraid that going this way instead of creating an ignore words list will create the need to edit .codespell-exclude-lines from time to time which from the POV of the contributors may be not worth the frustration and extra work :( I'd say codespell is here to help the maintainers and reviers (us), not the contributors, so we shouldn't put that on them.
I'd say we can live with occasional typos like 'ro' but not with crazy-from-the-contributors-pov workflow. How about we implement a more relaxed solution like https://github.com/protocolbuffers/protobuf/actions/runs/277120884/workflow?

Reply:
Wouldn't there will be the need to edit the `ignore_words_list` from time to time? If a contributor opens a PR with a valid word which codespell will recognize as an error, then they would have to adjust that. I created the `codespell-exclude-lines` file using a script, maybe that could be an option.

Reply:
`ignore_words_list` maintenance is leapp's team responsibility and it should be updated a lot less frequently as this solution will create false negatives for spell checking. Also it's an "add once - stays forever". Like with the `fo` example, you will need one rule to cover all further usages of `fo` as a file object.

`.codespell-exclude-lines` moves the responsibility to the contributor and it's an "add once - will fix only one exact occurence" solution. That's why you had to create 3 rules for using `fo`, and every new patch that has `fo` as a file object will need to add another one.

I am really in favor of cutting the contributors some slack here. But if you are unsure - we could definitely put that to the vote, that's what we have cabal meetings for :)

Reply:
Like with the `fo` example, if `fo` is in `ignore_words_list` and there is a `fo` typo in a PR, then it is back on reviewers to find that typo, which is why the spell checking is being introduced in the first place. 

However I agree that this would probably happen very rarely.

Reply:
Exactly, it's up to the reviewers - it's our job :) And we should not make it the contributors PITA. 
Also even if we as the reviewers miss it - it won't be the end of the world and nothing will really break.

Don't get me wrong - I absolutely love this patch, coming up with this idea was brilliant and it's a great addition to the existing checks we have. My concerns are only about the perfectness / contribution-friendly tradeoff.

Reply:
I think you are right. It is actually @pirat89's idea, so maybe he could give us his opinion?

Reply:
Do you think ignoring the word `ect` because of this single line: `FILES = ['/file1', '/file2', '/dir/ect/ory/file3', '/file4', '/file5']` is a good idea? I agree with the `ro` and `fo` examples, but I am not really sure about this one. I would rather change the path (as it is just a test).

Also similar example on this line: `Flags:                           esan3 zarch stfle msa ldisp eimm dfp edat etf3eh highgprs te vx vxd vxe gs vxe2 vxp sort dflt sie`, the offending word is `sie` which is the only valid occurrence in the entire project.

I think a combination of exclude lines file and `ignore_words`, might be the best solution, what do you think?

Reply:
Yeah, I think having a combination of both is just perfect.